websites/#/라프텔/presence.ts

```diff
@@ -0,0 +1,209 @@
+const presence = new Presence({
+	clientId: "1133347072200949770",
+});
+
+let prevData = "",
+	animeData: detail = {},
+	animeDataEpisode: episode = {};
+
+type detail = {
+	id?: string;
+	name?: string;
+	img?: string;
+	isEnding?: boolean;
+	animationInfo?: {
+		airYearQuarter?: string;
+	};
+	metaInfo?: {
+		avgRating?: string;
+	};
+};
+
+type episode = {
+	id?: string;
+	title?: string;
+	subject?: string;
+	episodeNum?: string;
+};
+
+type Obj<T> = {
+	[key: string]: T;
+};
+
+function convertCamel<T>(obj: Obj<unknown>): Obj<T> {
```

Comment:
I'd say this is rather unnecessary. You should just use eslint ignore on this rule

Reply:
The eslint ignore may be unnecessary as it only affects Deepscan, which means that the rule may need to be added like this:
https://github.com/PreMiD/Presences/blob/fcc689d2709c70084758151c7ee00c7822c6ac5c/websites/B/Board%20Game%20Arena/util.ts#L1-L25

We might be able to merge it in and ignore those specific Deepscan issues.

Reply:
thanks, fix it